wurst/_items/GemOfKnowledge.wurst

```diff
@@ -0,0 +1,18 @@
+package GemOfKnowledge
+
+import ItemObjEditing
+
+@compiletime function create_gemofknowledge()
+    new ItemDefinition('I03F', 'bspd')
+    ..setName("Gem of Knowledge")
+    ..setAbilities("A050,AIx1")
+    ..setTooltipBasic("Trade for Gem of Knowledge")
+    ..setHotkey("")
+    ..setTooltipExtended("Increase the growth rate of the wearer. Does not work for Beast Master, Hunter")
+    ..setDroppedWhenCarrierDies(true)
+    ..setLumberCost(40)
+    ..setGoldCost(0)
+    ..setInterfaceIcon("ReplaceableTextures\\CommandButtons\\BTNEnchantedGemstone.blp")
+    ..setDescription("Increases Growth Rate")
+    ..setLevel(0)
+    ..setCanBeSoldToMerchants(false)
```

Comment:
Do you know the default values for any of these? I think in general we'll want to write out some default values but others may just be noise. For example, if `setDroppedWhenCarrierDies` is true by default, we wouldn't want to write that out for every item.

Reply:
I tested it with false value (just checking if it worked properly), still dropped when the carrier died